src/System.Xml.ReaderWriter/tests/XmlWriter/System.Xml.RW.XmlWriter.Tests.csproj

```diff
@@ -29,6 +29,10 @@
       <Project>{be8ed8c1-c314-4c4e-a929-64c9c8b3552a}</Project>
       <Name>XunitTraitsDiscoverers</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\System.Text.Encoding.CodePages\src\System.Text.Encoding.CodePages.csproj">
```

Comment:
I'd rather change the infra to always reference the newest version. Waiting doesn't seem like a viable solution to me, no one will remember to update the tests in a month.